integrations/index.md

```diff
@@ -21,21 +21,21 @@ Some of the most in-demand integrations are listed below.
 
 |                                                                Name                                                                 | Description                                                               |
 |:-----------------------------------------------------------------------------------------------------------------------------------:|---------------------------------------------------------------------------|
-| [Auth.js][auth-js] | Implement authentication and authorization for web applications.          |
-|                                                           [Auth0][auth0]                                                            | Securely manage user authentication and access controls for applications. |
-|                                                            [Okta][okta]                                                             | Secure authentication and user identity management for applications.      |
+| <img isIcon src='https://assets.timescale.com/docs/icons/auth-logo.png' alt='auth-logo'  />[Auth.js][auth-js] | Implement authentication and authorization for web applications.          |
```

Comment:
It would be nice to expose some page, maybe under `assets.timescale.com/docs/icons` where one could see a preview of available icons

Reply:
Hi Lucas, just wondering what the use case is for this.

Reply:
This way whenever someone needs to add one you can check the page for a reference of available logos and their urls since we're hosting them (like the variables page)